content/blog/2018-06-07-when-to-use-derived-state.md

```diff
@@ -0,0 +1,327 @@
+---
+title: "When should I use derived state?"
```

Comment:
Nit: All Capitalized Please

Reply:
I'd also prefer “When to Use Derived State?” because I think a personal-sounding title looks a bit odd in the list. Like someone posted their question on our blog. :-)

<img width="443" alt="screen shot 2018-06-05 at 6 40 34 pm" src="https://user-images.githubusercontent.com/810438/40992914-fb222bc4-68ef-11e8-856d-302e753d8f0e.png">

Reply:
Oh, whoops. I mean to rename the title to match the file (which was already "when to use derived state"). Good call.